luxtronik/discover.py

```diff
@@ -32,35 +34,37 @@ def discover() -> list[(str, int)]:
 
         # send AIT magic broadcast packet
         server.sendto(LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode(), ("<broadcast>", port))
-        LOGGER.debug("Sending broadcast request %s", LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode())
+        LOGGER.debug(
+            "Sending broadcast request %s", LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode()
+        )
 
         while True:
             try:
-                res, con = server.recvfrom(1024)
-                res = res.decode("ascii", errors="ignore")
+                recv_bytes, con = server.recvfrom(1024)
+                res = recv_bytes.decode("ascii", errors="ignore")
                 # if we receive what we just sent, continue
                 if res == LUXTRONIK_DISCOVERY_MAGIC_PACKET:
                     continue
                 ip_address = con[0]
                 # if the response starts with the magic nonsense
                 if res.startswith(LUXTRONIK_DISCOVERY_RESPONSE_PREFIX):
-                    res = res.split(";")
```

Comment:
mypy: error: Incompatible types in assignment (expression has type "List[bytes]", variable has type "bytes")  [assignment]
Double use of var res